packages/edit-site/src/components/page-pages/default-views.js

```diff
@@ -28,7 +28,7 @@ const DEFAULT_PAGE_BASE = {
 };
 
 const DEFAULT_VIEWS = {
-	page: [
+	'/pages': [
```

Comment:
Personally I feel this key is useless because that whole file is within page-pages.

Reply:
The key was removed 👍